torch/quantization/ns/graph_matcher.py

```diff
@@ -352,14 +360,30 @@ class GraphMatchingException(Exception):
 
 class SugraphTypeRelationship(enum.Enum):
     # same type
-    # example: F.linear and toq.linear, or nn.Conv2d and nn.Conv2d
+    # example: F.linear and F.linear, or nn.Conv2d and nn.Conv2d
```

Comment:
Is this a typo? SugraphType?

Reply:
yes, the next PR in the stack fixes this